dags/public-health/covid19/shelter-to-esri.py

```diff
@@ -5,13 +5,15 @@
 import os
 from datetime import datetime, timedelta
 
-import arcgis
-import geopandas as gpd
 import pandas as pd
 import pytz
 from airflow import DAG
 from airflow.hooks.base_hook import BaseHook
 from airflow.operators.python_operator import PythonOperator
+from airflow.utils.email import send_email
+
+import arcgis
```

Comment:
I think this import moving noise may be due to a different version of `isort` being installed than that listed in the pre-commit hooks?

Reply:
got it. Any idea on how to resolve?